lib/exercism/user_finished_tracks.rb

```diff
@@ -24,9 +24,19 @@ def started_tracks
     ActiveRecord::Base.connection.execute(completed_count_sql).to_a
   end
 
+  def count_finished_problems(user_finished_exercises, track)
+    (user_finished_exercises & track.problems.map(&:slug)).size
+  end
+
+  def completed?(track_attributes)
```

Comment:
@bernardoamc  
Also, I have a question about where to puts the methods, currently I put them under private, is that good enough ?